std/traits.d

```diff
@@ -248,44 +248,33 @@ private
 
     static if (is(ucent))
     {
-        alias CentTypeList         = TypeTuple!(cent, ucent);
-        alias SignedCentTypeList   = TypeTuple!(cent);
-        alias UnsignedCentTypeList = TypeTuple!(ucent);
+        alias CentTypeSeq         = AliasSeq!(cent, ucent);
+        alias SignedCentTypeSeq   = AliasSeq!(cent);
+        alias UnsignedCentTypeSeq = AliasSeq!(ucent);
     }
     else
     {
-        alias CentTypeList         = TypeTuple!();
-        alias SignedCentTypeList   = TypeTuple!();
-        alias UnsignedCentTypeList = TypeTuple!();
+        alias CentTypeSeq         = AliasSeq!();
+        alias SignedCentTypeSeq   = AliasSeq!();
+        alias UnsignedCentTypeSeq = AliasSeq!();
     }
 
-    alias IntegralTypeList      = TypeTuple!(byte, ubyte, short, ushort, int, uint, long, ulong, CentTypeList);
-    alias SignedIntTypeList     = TypeTuple!(byte, short, int, long, SignedCentTypeList);
-    alias UnsignedIntTypeList   = TypeTuple!(ubyte, ushort, uint, ulong, UnsignedCentTypeList);
-    alias FloatingPointTypeList = TypeTuple!(float, double, real);
-    alias ImaginaryTypeList     = TypeTuple!(ifloat, idouble, ireal);
-    alias ComplexTypeList       = TypeTuple!(cfloat, cdouble, creal);
-    alias NumericTypeList       = TypeTuple!(IntegralTypeList, FloatingPointTypeList);
-    alias CharTypeList          = TypeTuple!(char, wchar, dchar);
-}
-
-package
-{
-    /// Add specific qualifier to the given type T.
-    template MutableOf(T)     { alias MutableOf     =              T  ; }
+    alias IntegralTypeSeq      = AliasSeq!(byte, ubyte, short, ushort, int, uint, long, ulong, CentTypeSeq);
+    alias SignedIntTypeSeq     = AliasSeq!(byte, short, int, long, SignedCentTypeSeq);
+    alias UnsignedIntTypeSeq   = AliasSeq!(ubyte, ushort, uint, ulong, UnsignedCentTypeSeq);
+    alias FloatingPointTypeSeq = AliasSeq!(float, double, real);
+    alias ImaginaryTypeSeq     = AliasSeq!(ifloat, idouble, ireal);
+    alias ComplexTypeSeq       = AliasSeq!(cfloat, cdouble, creal);
+    alias NumericTypeSeq       = AliasSeq!(IntegralTypeSeq, FloatingPointTypeSeq);
+    alias CharTypeSeq          = AliasSeq!(char, wchar, dchar);
 }
 
-/// Add specific qualifier to the given type T.
+template MutableOf(T)     { alias MutableOf     =              T  ; }
 template InoutOf(T)       { alias InoutOf       =        inout(T) ; }
-/// ditto.
 template ConstOf(T)       { alias ConstOf       =        const(T) ; }
-/// ditto.
 template SharedOf(T)      { alias SharedOf      =       shared(T) ; }
-/// ditto.
 template SharedInoutOf(T) { alias SharedInoutOf = shared(inout(T)); }
-/// ditto.
 template SharedConstOf(T) { alias SharedConstOf = shared(const(T)); }
-/// ditto.
 template ImmutableOf(T)   { alias ImmutableOf   =    immutable(T) ; }
 
```

Comment:
does anyone know what @jamadagni was trying to do here?
AFAICT this change undocuments all these templates, doesn't it?